package.json

```
@@ -76,7 +76,7 @@
     "lint": "standard | snazzy",
     "lint:fix": "standard --fix | snazzy",
     "test": "node scripts/generate-pem && npm run test:tap && npm run test:node-fetch && npm run test:fetch && npm run test:cookies && npm run test:wpt && npm run test:websocket && npm run test:jest && npm run test:typescript",
-    "test:cookies": "tap test/cookie/*.js",
+    "test:cookies": "borp --coverage test/cookie/*.js",
```

```suggestion
    "test:cookies": "borp --coverage \"test/cookie/*.js\"",
```

This will fix windows

---

big question why this is needed for borp and not for tap

---

> big question why this is needed for borp and not for tap

I'll take a look at it